app/src/main/res/values-sv/strings.xml

```diff
@@ -1,256 +1,87 @@
-    <string name="alert">Alert</string>
-    <string name="alert_cloud_connection_description">Varning om sensordata inte har uppdaterats till molnet på mer än %1$,d minuter.</string>
-    <string name="alert_cloud_connection_dialog_description">Ange önskad fördröjning som ska användas i minuter innan larmet utlöses. Minsta värde är 2 minuter.</string>
-    <string name="alert_cloud_connection_dialog_title">Ställ in molnanslutningsvarning</string>
-    <string name="alert_cloud_connection_title">Molnuppkoppling</string>
-    <string name="alert_dialog_max">Max (%1$s)</string>
-    <string name="alert_dialog_min">Min (%1$s)</string>
-    <string name="alert_dialog_title_humidity">Ställ in fuktighetsalarm</string>
-    <string name="alert_dialog_title_pressure">Ställ in lufttryckssalarm</string>
-    <string name="alert_dialog_title_rssi">Ställ in signalstyrkealarm</string>
-    <string name="alert_dialog_title_temperature">Ställ in temperaturalarm</string>
     <string name="alert_movement">Rörelse</string>
     <string name="alert_movement_description">Alarmera om rörelse upptäcks</string>
     <string name="alert_notification_humidity_high">För hög Luftfuktighet!</string>
-    <string name="alert_notification_humidity_high_threshold">Luftfuktigheten är över %1$s</string>
     <string name="alert_notification_humidity_low">För låg Luftfuktighet!</string>
-    <string name="alert_notification_humidity_low_threshold">Luftfuktigheten är under %1$s</string>
     <string name="alert_notification_movement">Rörelse upptäckt!</string>
-    <string name="alert_notification_offline_message">Sensorn har varit offline över %1$s minuter</string>
     <string name="alert_notification_pressure_high">För högt Lufttryck!</string>
-    <string name="alert_notification_pressure_high_threshold">Lufttrycket är över %1$s</string>
     <string name="alert_notification_pressure_low">För lågt Lufttryck!</string>
-    <string name="alert_notification_pressure_low_threshold">Lufttrycket är under %1$s</string>
     <string name="alert_notification_rssi_high">För hög signalstyrka!</string>
-    <string name="alert_notification_rssi_high_threshold">Signalstyrkan är över %1$s</string>
     <string name="alert_notification_rssi_low">För låg signalstyrka!</string>
-    <string name="alert_notification_rssi_low_threshold">Signalstyrkan är under %1$s</string>
     <string name="alert_notification_temperature_high">För hög temperatur!</string>
-    <string name="alert_notification_temperature_high_threshold">Temperaturen är över %1$s</string>
     <string name="alert_notification_temperature_low">För låg temperatur!</string>
-    <string name="alert_notification_temperature_low_threshold">Temperaturen är under %1$s</string>
     <string name="alert_subtitle_off">Av</string>
-    <string name="alert_subtitle_on">Alarmera när mindre än %1$s eller mer än %2$s</string>
+    <string name="alert_subtitle_on">Alarmera när mindre än {%.0f^%1$s} eller mer än {%.0f^%2$s}</string>
```

Comment:
```suggestion
    <string name="alert_subtitle_on">Alarmera när mindre än %1$s eller mer än {%.0f^%2$s}</string>
```

Reply:
probably related to https://github.com/ruuvi/com.ruuvi.station/pull/1154/files#diff-737e9d7721ec30928f6223b6d9d16d3ee931c929ee65b38b13cf625db60901e5R30-R31

Reply:
candidate fix https://github.com/ruuvi/com.ruuvi.station/pull/1155/commits/fdeef055b1fddeaefc7eeedea860bb7754f3c833